code/modules/mob/status_procs.dm

```diff
@@ -118,6 +118,15 @@
 	..()
 	update_canmove()
 
+/////////////////////////////////// STASIS ///////////////////////////////////
+
+/mob/proc/IsInStasis()
+/mob/living/IsInStasis()
```

Comment:
return FALSE for the first

Reply:
Is `. =  FALSE` okay? (byond's `return` has weird overhead)

Reply:
resolved